Automattic/wp-calypso
# Remove social proof experiment (test variation)

### Changes proposed in this Pull Request

This PR removes the `jetpackSocialProofHeader`, and keeps the initial control variation.

### Testing instructions

- Download the PR and run both Calypso and cloud concurrently
- Verify that the experiment `jetpackSocialProofHeader` is no longer available in the a/b test menu
- Check that you see the correct header in the pricing page for Calypso, cloud, and connect (see captures below)

### Screenshots

_A/B test menu_
<img width="558" alt="Screen Shot 2021-03-23 at 4 03 16 PM" src="https://user-images.githubusercontent.com/1620183/112210498-22134700-8bf1-11eb-974b-9d8a3db63348.png">


_Calypso_ (`http://calypso.localhost:3000/plans/:site`)
<img width="1011" alt="Screen Shot 2021-03-23 at 4 03 45 PM" src="https://user-images.githubusercontent.com/1620183/112210544-35261700-8bf1-11eb-9b2c-8e4f2b58254e.png">


_Cloud_ (`http://jetpack.cloud.localhost:3001/pricing`)
<img width="1070" alt="Screen Shot 2021-03-23 at 4 05 27 PM" src="https://user-images.githubusercontent.com/1620183/112210737-6ef71d80-8bf1-11eb-8ece-9798ede38a2a.png">


_Connect_ (`http://calypso.localhost:3000/jetpack/connect/store`)
<img width="479" alt="Screen Shot 2021-03-23 at 4 05 59 PM" src="https://user-images.githubusercontent.com/1620183/112210786-82a28400-8bf1-11eb-957e-bab87bbfdca5.png">
